openedx/features/discounts/tests/test_views.py

```diff
@@ -0,0 +1,39 @@
+"""Tests of openedx.features.discounts.views"""
+# -*- coding: utf-8 -*-
+import jwt
+
+from django.test.client import Client
+
+from xmodule.modulestore.tests.factories import CourseFactory
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from student.tests.factories import UserFactory
+from rest_framework.test import APIRequestFactory
+
+from openedx.features.discounts.views import CourseUserDiscount
+
+
+class TestCourseUserDiscount(ModuleStoreTestCase):
+    """
+    CourseUserDiscount should return a jwt with the information if this combination of user and
+    course can receive a discount, and how much that discount should be.
+    """
+
+    def setUp(self):
+        super(TestCourseUserDiscount, self).setUp()
+        self.user = UserFactory.create()
+        self.course = CourseFactory.create(run='test', display_name='test')
+        self.client = Client()
+        self.client.login(username=self.user.username, password='test')
+
+    def test_course_user_discount(self):
+        """
+        Test that the api returns a jwt with the discount information
+        """
+        import pdb; pdb.set_trace()
```

Comment:
You left debugging in, so jenkins is failing.

Reply:
I know... :)